src/nvim/memory.c

```diff
@@ -92,6 +100,12 @@ void *xmalloc(size_t size)
   return ret;
 }
 
+/// free wrapper that returns delegates to the backing memory manager
+void xfree(void *ptr)
+{
+  free(ptr);
```

Comment:
What's the purpose of `xfree()`, if we redefine `free()`? Is the plan to call `je_*` explicitly from the `xfoo` family, in the future?